cmd/corimDisplay.go

```diff
@@ -61,72 +61,111 @@ func checkCorimDisplayArgs() error {
 	return nil
 }
 
-func display(signedCorimFile string, showTags bool) error {
+func display(corimFile string, showTags bool) error {
 	var (
-		signedCorimCBOR []byte
-		metaJSON        []byte
-		corimJSON       []byte
-		err             error
-		s               corim.SignedCorim
+		corimCBOR []byte
+		err       error
 	)
 
-	if signedCorimCBOR, err = afero.ReadFile(fs, signedCorimFile); err != nil {
-		return fmt.Errorf("error loading signed CoRIM from %s: %w", signedCorimFile, err)
+	// read the CoRIM file
+	if corimCBOR, err = afero.ReadFile(fs, corimFile); err != nil {
+		return fmt.Errorf("error loading CoRIM from %s: %w", corimFile, err)
 	}
 
-	if err = s.FromCOSE(signedCorimCBOR); err != nil {
-		return fmt.Errorf("error decoding signed CoRIM from %s: %w", signedCorimFile, err)
-	}
+	// try to decode as a signed CoRIM
+	var s corim.SignedCorim
+	if err = s.FromCOSE(corimCBOR); err == nil {
+		// successfully decoded as signed CoRIM
```

Comment:
The code in this branch should be in a displaySignedCorim function to avoid the rightward drift

Reply:
@deeglaze okiee should i create separate functions for signed and unsigned corims and call them in display function?

Reply:
That’s my preference

Reply:
okieee @deeglaze i will update this file. Any more suggestions?

Reply:
Today I just started my holiday break. Splitting up into functions is a fine enough improvement for now.

Reply:
thanks @deeglaze really appropriate you suggestion...

Reply:
@deeglaze i have made some adjustments to the code according to your recommendations can you review them...